packages/patternfly-4/react-virtualized-extension/src/components/Virtualized/__snapshots__/VirtualizedTable.test.tsx.snap

```diff
@@ -0,0 +1,7975 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Actions virtualized table 1`] = `
```

Comment:
These huge snapshots are pretty unwieldy, it's hard to tell exactly what they are trying to assert. These tests are basically just "it renders correctly" tests, which is fine, but any little code change will cause these to fail and need updated snapshots, so it's up to the person updating the snapshots to look through them and determine if the change was bad or not.

I think that can be addressed as a future enhancement, these are better than no tests. In the future it might be nice to add some tests that use shallow rendering and assert some specific behavior of the code in reaction to props or state changing, or simulated user interaction. Those kinds of smaller, more targeted tests will be more likely to only fail if something is really wrong.